src/Hakyll/ShortcutLinks.hs

```diff
@@ -10,6 +10,21 @@ package in websites generated by [hakyll](https://hackage.haskell.org/package/ha
 
 The flexible interface allows to use the supported huge collection of shortcuts
 along with using custom ones.
+
+Here is a few examples of the `@github` shortcut:
+
+- Link to a user:
+  | Shortcut | Plain markdown |
+  | ------------------------------------- | ------------------------------------------------ |
+  | `[foo](@github)` | `[foo](https://github.com/foo)` |
+  | `[foo Github profile](@github(foo))` | `[foo Github profile](https://github.com/foo)` |
+
+- Link to a repository:
+
+  | Shortcut                                                   | Plain markdown                                                               |
+  | ---------------------------------------------------------- | ---------------------------------------------------------------------------- |
+  | `[bar](@github:foo)`                | `[bar](https://github.com/foo/bar)` |
+  | `[Github Source](@github(foo):bar)` | `[Github Source](https://github.com/foo/bar)`         |
```

Comment:
Unfortunately, table's syntax for Haddock is a bit different. You can see it here:

* https://haskell-haddock.readthedocs.io/en/latest/markup.html#grid-tables

Reply:
Okay, i'll fix that